lib/datasource/go/digest.spec.ts

```diff
@@ -49,16 +49,23 @@ describe('datasource/go/digest', () => {
       expect(res).toBeNull();
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
-    it('gitlab digest is not supported at the moment', async () => {
+    it('supports gitlab digest', async () => {
       httpMock
         .scope('https://gitlab.com/')
-        .get('/golang/text?go-get=1')
-        .reply(200, '');
+        .get('/group/subgroup?go-get=1')
+        .reply(200, loadFixture('go-get-gitlab.html'));
+      httpMock
+        .scope('https://gitlab.com/')
+        .get('/api/v4/projects/group%2Fsubgroup/repository/commits?per_page=1')
+        .reply(200, [{ id: 'abcdefabcdefabcdefabcdef' }]);
       const res = await getDigest(
-        { lookupName: 'gitlab.com/golang/text' },
+        { lookupName: 'gitlab.com/group/subgroup' },
         null
       );
-      expect(res).toBeNull();
+      expect(res).toMatchSnapshot();
```

Comment:
```suggestion
```

The `toBe` will be enough, so no snapshot terst required

Reply:
What about
```
expect(res).not.toBeNull();
expect(res).toBeDefined();
```
Ar they needed? `expect(res).toBe('abcdefabcdefabcdefabcdef');` should actually be sufficient for this test I think? (I've seen multiple expectations in other places so I did that too...)

Reply:
I removed these expectations too